Source/SwiftyDropbox/Platform/SwiftyDropbox_iOS/OAuthMobile.swift

```diff
@@ -32,13 +41,17 @@ extension DropboxClientsManager {
     /// - Parameters:
     ///     - sharedApplication: The shared UIApplication instance in your app.
     ///     - controller: A UIViewController to present the auth flow from.
+    ///     - loadingStatusDelegate: An optional delegate to handle loading experience during auth flow.
+    ///       e.g. Show a looading spinner and block user interaction while loading/waiting.
+    ///       If a delegate is not provided, the SDK will show a default loading spinner when necessary.
     ///     - openURL: Handler to open a URL.
     ///     - scopeRequest: Contains requested scopes to obtain.
     public static func authorizeFromControllerV2(
-        _ sharedApplication: UIApplication, controller: UIViewController?, openURL: @escaping ((URL) -> Void), scopeRequest: ScopeRequest?
+        _ sharedApplication: UIApplication, controller: UIViewController?, loadingStatusDelegate: LoadingStatusDelegate?, openURL: @escaping ((URL) -> Void), scopeRequest: ScopeRequest?
```

Comment:
is this delegate ever non nil? i see the usage here passes in a nil instance https://github.com/dropbox/SwiftyDropbox/pull/271/files#diff-47b40edc806c91305e9791514ebf7eceR24

Reply:
3rd party app could set the delegate so they could have control over the UX of loading.
in the sample app, I passed in nil to simply use the default implementation.